lib/Operation.php

```diff
@@ -69,12 +80,7 @@ public function __construct(ISystemTagObjectMapper $objectMapper, ISystemTagMana
 	 * @param string $file
 	 */
 	public function checkOperations(IStorage $storage, $fileId, $file) {
-		$this->checkManager->setFileInfo($storage, $file);
```

Comment:
@nickvergessen Not sure if using the regular file events causes issues here or if we should keep the CacheUpdateEvent/CacheInsertEvent. I guess there were issues with object storage using the storage wrapper according to https://github.com/nextcloud/files_automatedtagging/pull/71 ?

Reply:
Yeah and also files:scan I think. I'd prefer to leave this untouched as much as possible.

Reply:
@blizzz I guess we need to move the automated tagging to an IComplexOperation then as well, or do we have a way to limit the available events for an ISpecificOperation?

Reply:
@juliushaertl yes, automatted tagging must be an `IComplexOperation` because the tags are required to be set asap due to integration with .e.g files_accesscontrol